Bridges/HttpKernel.php

```diff
@@ -177,10 +157,10 @@ protected function mapRequest(ReactRequest $reactRequest)
             $class = '\Symfony\Component\HttpFoundation\Request';
         }
 
-        $syRequest = new $class($query, $post, $attributes = [], $cookies, $files, $_SERVER, $reactRequest->getBody());
+        /** @var SymfonyRequest $syRequest */
+        $syRequest = new $class($query, $post, $attributes = [], $_COOKIE, $files, $_SERVER, $reactRequest->getBody());
```

Comment:
Would it be possiblr to create a new, clean pr and describe the problem it tries to solve? Must say that I'm totally lost by now.